src/libstd/net/parser.rs

```diff
@@ -372,6 +372,25 @@ impl FromStr for SocketAddr {
 /// [`IpAddr`], [`Ipv4Addr`], [`Ipv6Addr`], [`SocketAddr`], [`SocketAddrV4`], and
 /// [`SocketAddrV6`].
 ///
+/// # Potential causes:
+///
+/// `AddrParseError` may be thrown because the provided string does not parse as the given type,
```

Comment:
Sorry, forgot to ask you to turn this into an url (like done in a few places here too).

Reply:
The current page we are reading is the `AddrParseError` page. It seemed useless to me, but, ok, I'll apply the changes.

Reply:
Ah then my bad. Just ignore this comment.

Reply:
Too fast, I had applied the changes. :)